src/dex-ui/pages/dao/DAOProposals/Forms/DAOTokenAssociateDetailsForm.tsx

```diff
@@ -54,6 +56,23 @@ export function DAOTokenAssociateDetailsForm() {
         isInvalid={Boolean(errors?.description)}
         errorMessage={errors?.description && errors?.description?.message}
       />
+      {daoType !== Paths.DAOs.Multisig ? (
+        <FormInput<"linkToDiscussion">
```

Comment:
as mentioned above we can include `linkToDiscussion` n both DAO's Proposal

Reply:
Done.